data/strings/types_strings.txt

```diff
@@ -21376,7 +21377,7 @@
     sv = Stad
     sw = Jiji
     th = เมือง
-    tr = Şehir
+    tr = İl
```

Comment:
The source word is "City", so Şehir is probably the right translation?

Reply:
Checked again, [type.place.state] should be "il" for Turkey but it should be "Eyalet" for other countries. 

You're right, [type.place.city] should be "şehir" since it indicates city centers. But normally towns are also "şehir" in Turkish.

According to Turkish Language Association dictionary,
**Şehir**: A residential area with more than twenty thousand inhabitants, most of whom are engaged in commerce, industry, service or administrative activities, usually without agricultural activities.
Used for: [type.place.city]

**İl**: A part of the country governed by a governor; province.
Used for: [type.place.state]

**İlçe**: The department following the province in the division of the country in terms of administration; district governorship.
Used for: [type.place.town]